docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       retries: 3
     # UN-COMMENT for local testing:
     #ports:
-    #  - "5432:5432"
+      #- "5432:5432"
```

Comment:
Please revert this to the previous value. This sounds nit-picky,  but here's the reason: when looking through code diffs between commits, we want the "significant" changes to be called out. This change is not functionally different, so it is distracting when viewing the diffs.

Reply:
I'm not really sure what you mean by this one. I just uncommented it and then commented it again.

Reply:
Latest commit in master:

https://github.com/Doveps/mono/blob/fe2fb79bf1d01cd81f1abd6f4057dee03565942b/docker-compose.yml#L17-L18

Reply:
https://github.com/Doveps/mono/blob/fe2fb79bf1d01cd81f1abd6f4057dee03565942b/docker-compose.yml#L17-L18


That's the last commit from the master. I'm not really sure what values, though.